apps/api/src/controllers/v1/extract-status.ts

```diff
@@ -0,0 +1,53 @@
+import { Response } from "express";
+import {
+  supabaseGetJobByIdOnlyData,
+  supabaseGetJobsById,
+} from "../../lib/supabase-jobs";
+import { scrapeStatusRateLimiter } from "../../services/rate-limiter";
+import { RequestWithAuth } from "./types";
+
+export async function extractStatusController(
+  req: RequestWithAuth<{ jobId: string }, any, any>,
+  res: Response,
+) {
+  try {
```

Comment:
You must not wrap v1 endpoint controllers in a try-catch. Error handling is done on the Express middleware level.